### PR TITLE
Bumps the simulator version to make testing local changes easier

### PR DESCRIPTION
**Pull request by ndizazzo**
## Priority
Low

## What Changed & Why
This PR bumps the sim version and Travis config to use a newer version of Xcode.

NOTE: I was getting test output in different order during each run, maybe due to some underlaying change in how Xcode executes things. Sorting test outputs and reference data seemed to do the trick to correct this behaviour.

### CODE REVIEW

File: spec/testdumper_acceptance_spec.rb [hunk 1]
```diff
@@ -10,14 +14,34 @@
 {"test":"1","className":"iPhoneTestClassGama","event":"end-test","totalDuration":"0"}
 {"test":"1","className":"iPhoneTestClassOmega","event":"end-test","totalDuration":"0"}
 {"message":"Completed Test Dumper","event":"end-action","testType":"LOGICTEST"}
+{"message":"Starting Test Dumper","event":"begin-test-suite","testType":"LOGICTEST"}
+{"event":"begin-ocunit","bundleName":"SwiftTestTarget.xctest","targetName":"SwiftTestTarget"}
+{"test":"1","className":"ObjCTestClass","event":"end-test","totalDuration":"0"}
+{"test":"1","className":"SwiftTestTarget","event":"end-test","totalDuration":"0"}
+{"message":"Completed Test Dumper","event":"end-action","testType":"LOGICTEST"}
+{"message":"Starting Test Dumper","event":"begin-test-suite","testType":"APPTEST"}
+{"event":"begin-ocunit","bundleName":"iPadTestTarget.xctest","targetName":"iPadTestTarget"}
```
Comment (justinseanmartin) [inline spec/testdumper_acceptance_spec.rb:23]: This is interesting. Why is this newly showing up now?

Comment (ndizazzo) [reply spec/testdumper_acceptance_spec.rb:23]: I'm not really 100% certain without spending more time investigating. 

I would probably try to isolate this by progressively bumping the Xcode version from 9 to 10, then 10 to 11 to see which jump the issue started appearing in and work backwards from there, maybe checking out Xcode change logs for each version hop.

File: spec/testdumper_acceptance_spec.rb [hunk 2]
```diff
@@ -92,14 +116,14 @@ def test_xcknife_exemplar
...
+    expect(IO.read(outpath).lines.sort).to eq(EXPECTED_OUTPUT2.lines.sort)
   end
 
   it "dumps tests using nm on example project" do
     test_bundle_names = ["CommonTestTarget", "iPhoneTestTarget", "SwiftTestTarget"]
     expect_any_instance_of(XCKnife::TestDumperHelper).to receive(:list_tests_with_nm).exactly(3).times.and_call_original
     expect_any_instance_of(XCKnife::TestDumperHelper).to receive(:list_tests_with_simctl).once.and_call_original
     expect { XCKnife::TestDumper.new([derived_data_path, outpath, simulator_uuid, '--naive-dump', test_bundle_names.join(',')], logger: logger).run }.not_to raise_error
-    expect(IO.read(outpath).lines).to eq(EXPECTED_OUTPUT.lines)
+    expect(IO.read(outpath).lines.sort).to eq(EXPECTED_OUTPUT.lines.sort)
```
Comment (justinseanmartin) [inline spec/testdumper_acceptance_spec.rb:127]: I'd explicitly call this `EXPECTED_NAIVE_DUMP_OUTPUT` if the behavior is different between naive and normal test dump modes, though we're not doing anything AFAIK that should lead to different results.

Comment (ndizazzo) [reply spec/testdumper_acceptance_spec.rb]: This stopped happening when I upped things to 11.6, and set the missing sim-runtime to 11-6.  ¯\\_(ツ)_/¯

File: spec/testdumper_acceptance_spec.rb (hunk 1 above)
Comment (ndizazzo) [reply spec/testdumper_acceptance_spec.rb]: I think this was caused by a missed update on sim_runtime. Seem to behave correctly now.

**Pull request merged**